website/payments/api/v2/admin/views.py

```diff
@@ -0,0 +1,144 @@
+import rest_framework.filters as framework_filters
```

Comment:
<img width="897" alt="image" src="https://user-images.githubusercontent.com/7915741/121336820-2b709d80-c91c-11eb-9c4a-088d678d2cc3.png">

Swagger doesn't like it that the admin api views and the regular api views have the same name

Reply:
and it's also a bit confused by it, as it thinks a delete is allowed

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/7915741/121337080-71c5fc80-c91c-11eb-8f2e-b7c974411c19.png">